### PR TITLE
 AML-2021 Added pending transfer connection invitations to the transfers page

### PR DESCRIPTION
**Pull request by elliotritchie**

### CODE REVIEW

File: src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs [hunk 1]
```diff
@@ -48,17 +48,51 @@ public TransferConnectionInvitationRepository(EmployerApprenticeshipsServiceConf
                 var parameters = new DynamicParameters();
 
                 parameters.Add("@id", id, DbType.Int64);
-                parameters.Add("@status", TransferConnectionInvitationStatus.Sent, DbType.Int16);
+                parameters.Add("@status", TransferConnectionInvitationStatus.Pending, DbType.Int16);
+
+                const string sql = @"
+                    SELECT i.*, r.name AS ReceiverAccountName
```
Comment (iangledhill) [inline src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs]: This code cannot guarantee the order or layout of the TransferConnectionInvitation table. The i.* makes the code sensitive to schema changes.

File: src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs [hunk 2]
```diff
@@ -48,17 +48,51 @@ public TransferConnectionInvitationRepository(EmployerApprenticeshipsServiceConf
...
+        {
+            var result = await WithConnection(async c =>
+            {
+                var parameters = new DynamicParameters();
+
+                parameters.Add("@senderAccountId", senderAccountId, DbType.Int64);
+                parameters.Add("@status", TransferConnectionInvitationStatus.Pending, DbType.Int16);
+
+                const string sql = @"
+                    SELECT i.*, r.Name AS ReceiverAccountName
```
Comment (iangledhill) [inline src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs]: i.*

File: src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs [hunk 3]
```diff
@@ -69,7 +103,7 @@ public TransferConnectionInvitationRepository(EmployerApprenticeshipsServiceConf
                 parameters.Add("@receiverAccountId", receiverAccountId, DbType.Int64);
 
                 return await c.QueryAsync<TransferConnectionInvitation>(
-                    sql: "SELECT * FROM [employer_account].[TransferConnectionInvitation] WHERE SenderAccountId = @senderAccountId AND ReceiverAccountId = @receiverAccountId;",
+                    sql: "SELECT * FROM [employer_account].[TransferConnectionInvitation] WHERE SenderAccountId = @senderAccountId AND ReceiverAccountId = @receiverAccountId",
```
Comment (iangledhill) [inline src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/TransferConnectionInvitationRepository.cs]: * - don't know if this is our teams/DAS convention?

**Comment by elliotritchie** [comment]
Merged with #1132.

**Pull request closed without merging**